src/components/twoColumnLayout/TwoColumnLayout.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+
+interface TwoColumnLayoutProps {
+  left: React.ReactNode;
+  right: React.ReactNode;
+  className?: string;
+}
+
+const TwoColumnLayout: FC<TwoColumnLayoutProps> = ({ left, right, className }) => (
```

Comment:
You can satisfy the prop-types lint rule by supplying the type for the object you're destructuring:
```ts
{ left, right, className }: TwoColumnLayoutProps
```

(This makes the line too long so it has to look like this:
```suggestion
const TwoColumnLayout: FC<TwoColumnLayoutProps> = ({
  left,
  right,
  className,
}: TwoColumnLayoutProps) => (
```

Reply:
Agree, but was hoping not to do this since it should be inferred from `FC<TwoColumnLayoutProps>`